migrations/versions/0847_insert_new_callback_types.py

```diff
@@ -0,0 +1,22 @@
+"""
+Create Date: 2025-01-17 12:30:50.050539
+"""
+
+from alembic import op
+
+revision = "0847_insert_new_callback_types"
+down_revision = "0486_letter_rates_feb_2025"
+
+
+def upgrade():
+    insert_returned_letter_callback_type = "INSERT INTO service_callback_type VALUES ('returned_letters')"
+    insert_inbound_sms_callback_type = "INSERT INTO service_callback_type VALUES ('inbound_sms')"
+    op.execute(insert_returned_letter_callback_type)
+    op.execute(insert_inbound_sms_callback_type)
+
+
+def downgrade():
+    delete_returned_letter_callback_type = "DELETE FROM service_callback_type WHERE name = 'returned_letters'"
+    delete_inbound_sms_callback_type = "DELETE FROM service_callback_type WHERE name = 'inbound_sms'"
```

Comment:
This downgrade will start to fail as soon as we have any callbacks using these types. Should we also delete those first?

Reply:
Related, is the downgrade test script currently working?

Reply:
> This downgrade will start to fail as soon as we have any callbacks using these types. Should we also delete those first?

Good catch. We definitely don't want that, so I have updated the downgrade script to do nothing.

Reply:
> Related, is the downgrade test script currently working?

It works locally currently because no callbacks are using those types at the moment. However I have now updated the script.